templates/about/editorialPolicies.tpl

```diff
@@ -113,6 +117,30 @@
 </div>
 {/if}
 
+{if $currentJournal->getSetting('publishingMode') == $smarty.const.PUBLISHING_MODE_SUBSCRIPTION && $paymentConfigured && $currentJournal->getSetting('journalPaymentsEnabled') && $currentJournal->getSetting('acceptSubscriptionPayments') && $currentJournal->getSetting('purchaseIssueFeeEnabled') && $currentJournal->getSetting('purchaseIssueFee') > 0}
+<div id="purchaseIssue"><h3>{translate key="about.purchaseIssue"}</h3>
```

Comment:
@mfelczak, could you reformat this part with XML-style indentation? It's more readable IMO.